resources/dicts/patrols/beach/med/greenleaf.json

```diff
@@ -1314,10 +1314,10 @@
     "antagonize_fail_text": null
 },
 {
-    "patrol_id": "bch_med_gatheringraspberrysolo_greenleaf2",
+    "patrol_id": "bch_med_gatheringraspberrysoloapp_greenleaf1",
```

Comment:
you'll want to remove the season indicators from these so that the patrol sprite will still work